internal/configs/configs.go

```diff
@@ -95,7 +95,7 @@ type (
 
 func LoadMainConfig() (*Project, error) {
 	confLoader, err := configuro.NewConfig(
-		configuro.WithLoadFromConfigFile("./config.yaml", true),
+		configuro.WithLoadFromConfigFile("./examples/Django/emptyapi/config.yaml", true),
```

Comment:
WTF, bro?

Reply:
For this integration i created individual config and the project should have been loaded with this config.

Reply:
And you thought you should hardcode the path here?

Reply:
I'll change it myself